laserstudio/widgets/toolbars/markerslistdialog.py

```diff
@@ -0,0 +1,132 @@
+from PyQt6.QtCore import Qt
+from PyQt6.QtGui import QColor
+from PyQt6.QtWidgets import QPushButton, QDialog, QTreeWidget, QTreeWidgetItem, QVBoxLayout, QHBoxLayout
+from ..viewer import Viewer
+from ..marker import Marker
+
+class MarkerGroupListItem(QTreeWidgetItem):
+    def __init__(self, parent: QTreeWidget):
+        super().__init__(parent)
+        self._number_of_checked = 0
+
+    @property
+    def number_of_checked(self):
+        return self._number_of_checked
+    
+    @number_of_checked.setter
+    def number_of_checked(self, value):
+        self._number_of_checked = value
```

Comment:
There is currently no benefit in having this property, you can simplify the code by removing the getter and setter and renaming `_number_of_checked` to `number_of_checked`.

Reply:
It was the first intention, but it has been revised afterwards